library/general/src/modules/Arch.rb

```diff
@@ -221,6 +221,9 @@ def board_compatible
         # Cell and Maple based boards have no CHRP in /proc/cpuinfo
         # Pegasos and Cell do have CHRP in /proc/cpuinfo, but Pegasos2 should no be handled as CHRP
         # Efika is handled like Pegasos for the time being
+        # Treat PowerNV as CHRP. It is harmless for now. Patch for hwinfo is sent but it is better to be safe
+        if @_board_compatible == "PowerNV"
+          @_board_compatible = "CHRP"
```

Comment:
here is missing `end` or elsif for next if. This way it do not pass travis build ( hard requirement to merge request ).